### ISSUE TITLE
Xcode 10.2 - beta not work

### ISSUE THREAD

**Issue by JadianZheng** [XVimProject/XVim2#217]
Same to the title.

**Comment by pebble8888** [comment]
Thank you for your report.
Please try `develop` branch.

**Comment by Guardiannw** [comment]
Hey @pebble8888, getting this error in the `develop` branch: 

```
   'IDEKit/NSTextViewDelegate-Protocol.h' file not found
#import <IDEKit/NSTextViewDelegate-Protocol.h>
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by pebble8888** [comment]
@Guardiannw 
I have no build error at 7993a9d.

In my environment, 
ls -l /develop/XVim2/XVim2/XcodeHeader/IDEKit/NSTextViewDelegate-Protocol.h 
-rw-r--r--@ 1 pebble8888  staff  4209  3 21 20:33 /develop/XVim2/XVim2/XcodeHeader/IDEKit/NSTextViewDelegate-Protocol.h

`Header Search Paths` in xcode project is
$(SOURCE_ROOT)/XVim2/XcodeHeader .

$(SOURCE_ROOT) is `/develop/XVim2`

**Comment by ryantstone** [comment]
Can confirm I'm receiving the same error. And I checked the directory and `NSTextViewDelegate-Protocol.h` is not present in `IDEKit` it's present in ` XVim2/XcodeHeader/DVTKit` though. Below is a gist with the output from running make.

https://gist.github.com/ryantstone/7b52efcb5e468ffbe893d1a67d1aa501

**Comment by pebble8888** [comment]
Sorry for your inconvenience.
I got a cause. I'll fix 8 hours later.

**Comment by ryantstone** [comment]
That has fixed the problem for me. Thank you for helping maintain this project @pebble8888 it's something I use everyday and without it my work day would be much slower.

**Comment by hkjels** [comment]
Now that `10.2` is released, e10eab6 should probably be made more accessible

**Comment by JadianZheng** [comment]
@pebble8888 Thank you for your fix, it helps me a lot.
